src/dialect/snowflake.rs

```diff
@@ -214,6 +216,16 @@ impl Dialect for SnowflakeDialect {
     fn supports_show_like_before_in(&self) -> bool {
         true
     }
+
+    fn is_reserved_for_identifier(&self, kw: Keyword) -> bool {
```

Comment:
oh are we missing tests for the new behavior (I can't seem to find them in the current PR if so)?